src/components/install.tsx

```diff
@@ -1,8 +1,8 @@
-import React, {useEffect, useState} from "react"
+import React from "react"
 import CodeBlock from "@theme/CodeBlock"
 import Version from "./Version"
 
-const InstallCommand = () => {
+const InstallCommand = (): JSX.Element => {
```

Comment:
```suggestion
const InstallCommand = () => {
```
let typescript infer the type where ever  possible. If you enable lint rule for no implicit any it will not infer anything to any

Reply:
Do you want me change this in all the places, because we have added this return type for all the components and pages @amitksingh1490 .

Reply:
let it be